tutorials/plot_object_epochs.py

```diff
@@ -114,8 +117,9 @@
 
 ###############################################################################
 # Later on you can read the epochs with :func:`mne.read_epochs`. For reading
-# EEGLAB epochs files see :func:`mne.read_epochs_eeglab`.
-epochs = mne.read_epochs(epochs_fname)
+# EEGLAB epochs files see :func:`mne.read_epochs_eeglab`. We can also use
```

Comment:
Sure the ref to the EEGLAB reader here is appropriate? I'd expect it under IO, or the doc specifically dedicated to EEGLAB migration.